Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -897,11 +897,10 @@ def prepTextProperty(p, winSize, to="default", tt="default", cmap=None,
     if isinstance(tt, str):
         tt = vcs.elements["texttable"][tt]
 
-    if cmap is None:
-        if tt.colormap is not None:
-            cmap = tt.colormap
-        else:
-            cmap = 'default'
+    if tt.colormap is not None:
```

Comment:
why not just do this: 

if tt.colormap is not None:
  cmap = ***
else:
  cmap = +++